django/db/backends/sqlite3/features.py

```diff
@@ -18,7 +18,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     atomic_transactions = False
     can_rollback_ddl = True
     can_create_inline_fk = False
-    supports_paramstyle_pyformat = False
+    supports_paramstyle_pyformat = True
```

Comment:
Ah I assume this is why tests were passing on SQLite as well?

Reply:
@charettes Yes! There is a test called [test_pyformat_params](https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/tests/raw_query/tests.py#L231) with a `skipUnlessDBFeature` which was causing the test to be skipped. 

I’ll also make an update to the docs (once I am able to incorporate the feedback from the issue into the solution 😁)